imagery/i.landsat.toar/landsat_set.c

```diff
@@ -45,7 +45,10 @@ void sensor_TM(lsat_data * lsat)
     double wmax[] = { 0.52, 0.60, 0.69, 0.90, 1.75, 12.50, 2.35 };
     /* 30, 30, 30, 30, 30, 120 original, 60 resamples before Feb 25, 2010 and 30 after, 30 */
 
-    if (!lsat->sensor)
+    /* address of array 'lsat->sensor' will always evaluate to 'true' */
+    /* In this case the following condition is always false and */
+    /* strcpy() is never reached */
+    if (!*lsat->sensor)
```

Comment:
Looking at the other `sensor_*()`fns, it should be
```
    strcpy(lsat->sensor, "TM");
```
the `if (!lsat->sensor)` before must be removed.

Reply:
Updated as suggested.